common/src/main/java/tachyon/LeaderInquireClient.java

```diff
@@ -83,6 +84,7 @@ public synchronized String getMasterAddress() {
                 leader = master;
               }
             }
+            LOG.info("The leader master: " + leader);
```

Comment:
Will this logging be too verbose? I think `getMasterAddress` gets called on every client connection?